lore/stores/base.py

```diff
@@ -3,6 +3,8 @@
 import hashlib
 import inspect
 
+import sqlalchemy
```

Comment:
sqlalchemy is not a universal dependency, so it will need to be required here, similar to how it is done in lore.io.connection